### PR TITLE
Russian translations

### PR DESCRIPTION
**Pull request by dom1torii**
also fixed "en" to "sv" in cs2kz-styles.phrases.txt swedish translation (line 105)

### CODE REVIEW

**Review by zer0k-z — APPROVED** [review]

**Comment by zer0k-z** [comment]
thanks!

File: translations/cs2kz-timer.phrases.txt [hunk 1]
```diff
@@ -52,6 +55,7 @@
 		// Server: #1/24 Overall (-1:00.00) | #1/10 PRO (-2:00.00)
 		"#format"	"rank:d,max_rank:d,pb_diff_text:s,rank_pro:d,max_rank_pro:d,pb_diff_text_pro:s"
 		"en"		"{grey}Server: #{default}{rank}{grey}/{default}{max_rank}{grey} {yellow}Overall{grey}{pb_diff_text} | #{rank_pro}/{max_rank_pro} {blue}PRO{grey}{pb_diff_text_pro}"
+		"en"		"{grey}Сервер: #{default}{rank}{grey}/{default}{max_rank}{grey} {yellow}Среди всех{grey}{pb_diff_text} | #{rank_pro}/{max_rank_pro} {blue}PRO{grey}{pb_diff_text_pro}"
```
Comment (jvnipers) [inline translations/cs2kz-timer.phrases.txt:58]: "en" here too :D

Comment (dom1torii) [reply translations/cs2kz-timer.phrases.txt:58]: ah my bad
can i somehow edit this pull request or i need to make a new one?

**Comment by zer0k-z** [comment]
should already be fixed now.

**Pull request merged**